### DISCUSSION TITLE
how to convert chinese?

### DISCUSSION THREAD

**Discussion by haoisli9** [arnm/ob-mermaid#29] [General]
I am using emacs 29.0.50 in win10, and installed mermaid and ob-mermaid.
when using mermaid in org-mode, mermaid can convert english word correct, but chinese can not show in image.
maybe i miss some configuration?

thanks.

**Comment by arnm** [comment]
@haoisli9 

I will have to try examples of translations, do you happen to have some examples? I'm not sure if this would be related to mermaid-cli itself rather than this package specifically
